### PR TITLE
Updated OpenEdge keywords

### PR DESCRIPTION
**Pull request by movedoa**
I updated the OpenEdge ABL keyword list.
My data comes from the official documentation.
[Link to documentation](https://docs.progress.com/bundle/openedge-abl-reference-117/page/Keyword-Index.html)

### CODE REVIEW

**Comment by movedoa** [comment]
Not sure why the linter is failing, it seems to complain about 'X' but that was there before.
Any help would be appreciated.

File: pygments/lexers/_openedge_builtins.py [hunk 1]
```diff
@@ -591,9 +606,13 @@
     'DROP-DOWN-LIST',
     'DROP-FILE-NOTIFY',
     'DROP-TARGET',
+    'ds-close-cursor',
```
Comment (birkenfeld) [inline pygments/lexers/_openedge_builtins.py:609]: why is this lowercased?

File: pygments/lexers/_openedge_builtins.py [hunk 2]
```diff
@@ -400,6 +406,7 @@
     'CREATE-TEST-FILE',
     'CURRENT',
     'CURRENT_DATE',
+    'CURRENT_DATE',
```
Comment (birkenfeld) [inline pygments/lexers/_openedge_builtins.py:409]: this one is duplicated

File: pygments/lexers/_openedge_builtins.py [hunk 3]
```diff
@@ -1354,6 +1389,7 @@
     'MARK-NEW',
     'MARK-ROW-STATE',
     'MATCHES',
+    'MAX',
```
Comment (birkenfeld) [inline pygments/lexers/_openedge_builtins.py:1392]: max is duplicated below

File: pygments/lexers/_openedge_builtins.py [hunk 4]
```diff
@@ -1415,6 +1451,7 @@
     'MESSAGE-AREA-FONT',
     'MESSAGE-LINES',
     'METHOD',
+    'MIN',
```
Comment (birkenfeld) [inline pygments/lexers/_openedge_builtins.py:1454]: also duplicate below

File: pygments/lexers/_openedge_builtins.py [hunk 5]
```diff
@@ -448,6 +457,7 @@
     'DATA-SOURCE-MODIFIED',
     'DATA-SOURCE-ROWID',
     'DATA-TYPE',
+    'DATE',
```
Comment (birkenfeld) [inline pygments/lexers/_openedge_builtins.py:460]: this is not alphabetically sorted

**Review by birkenfeld — CHANGES_REQUESTED** [review]
Such a failure points to duplicate entries. I've marked some inline.

File: pygments/lexers/_openedge_builtins.py (hunk 3 above)
Comment (movedoa) [reply pygments/lexers/_openedge_builtins.py:1390]: Done

File: pygments/lexers/_openedge_builtins.py (hunk 5 above)
Comment (movedoa) [reply pygments/lexers/_openedge_builtins.py]: The whole list was not really 100% sorted correctly, i have fixed the sorting of the whole list now.

File: pygments/lexers/_openedge_builtins.py (hunk 1 above)
Comment (movedoa) [reply pygments/lexers/_openedge_builtins.py]: Its like that in the documentation, i fixed it anyway to stay consistent

File: pygments/lexers/_openedge_builtins.py (hunk 4 above)
Comment (movedoa) [reply pygments/lexers/_openedge_builtins.py:1451]: Done

File: pygments/lexers/_openedge_builtins.py (hunk 2 above)
Comment (movedoa) [reply pygments/lexers/_openedge_builtins.py]: Done

**Review by birkenfeld — APPROVED** [review]
thanks!

**Pull request merged**